tests/core/pyspec/eth2spec/test/helpers/custody.py

```diff
@@ -119,7 +119,7 @@ def get_valid_custody_response(spec, state, bit_challenge, custody_data, challen
     data_branch = get_merkle_proof(data_tree, chunk_index)
 
     bitlist_chunk_index = chunk_index // BYTES_PER_CHUNK
-    bitlist_chunks = chunkify(pack(bit_challenge.chunk_bits))
+    bitlist_chunks = None  # TODO chunkify(pack(bit_challenge.chunk_bits))
```

Comment:
What's going on here? Is this resolved in recent Phase 1 mods?

Reply:
This was one of the more tricky parts, but unused currently. The bitlist chunkification doesn't make much sense when the bitlist is already a tree. I can probably write some function in remerkleable to return the leaf nodes as chunks. But it was more of a pointless blocker than a critical thing for this PR.